src/test/java/br/com/academiadev/thunderpets/controller/PetControllerTest.java

```diff
@@ -257,13 +258,13 @@ public void dadoPets_quandoBuscoPorCidadeJoinvilleOrdenacaoPadrao_entaoBuscarPet
                 .param("cidade", "JOINVILLE"));
 
         //Entao
-        listaDePets.andExpect(status().isOk())
-                .andExpect(jsonPath("$.content[0].nome", is("Pocotó")))
-                .andExpect(jsonPath("$.content[0].localizacao.cidade", is("Joinville")))
-                .andExpect(jsonPath("$.content[0].localizacao.estado", is("Blumenal")))
-                .andExpect(jsonPath("$.content[1].nome", is("Carijó")))
-                .andExpect(jsonPath("$.content[1].localizacao.cidade", is("Joinville")))
-                .andExpect(jsonPath("$.content[1].localizacao.estado", is("Santa Catarina")));
+//        listaDePets.andExpect(status().isOk())
+//                .andExpect(jsonPath("$.content[0].nome", is("Pocotó")))

```

Comment:
O teste falhou porque a busca por Pets por padrão é feita por dataRegistro DESC, conforme `PetController`:

> @ApiParam(value = "Direção da ordenação: ascendente ou descendente")
  @RequestParam(defaultValue = "DESC") Sort.Direction direcao,
@ApiParam(value = "Nome da coluna que será usada para a ordenação")
  @RequestParam(defaultValue = "dataRegistro") String campoOrdenacao,

E, quando eu criei os testes eu adicionei valores fixos ao atributo dataRegistro, conforme `PetDTOUtil`:

> .nome("Carijó")
.dataRegistro(LocalDate.of(2018, 8,4))

> .nome("Pocotó")
.dataRegistro(LocalDate.of(2018, 12,1))

Desculpeee, não havia lembrado da auditoria dos pets quando fiz os testes. My bad 😐

Mas entãooo, acho que dá pra remover esse atributo lá do builder, igual você fez nas classes `PetMapperTest` e `PetUtil`. Logo, como no teste em questão o pet Carijó é criado antes, é só inverter. Dá pra criar o pet Pocotó antes ou inverter os índices dos `jsonPath` esperados.